components/api/constants/constants.bal

```diff
@@ -57,3 +59,11 @@ public const string UPDATED_DATE = "UPDATED_DATE";
 
 public const string CACHE_EXPIRY_VAR = "token.cache.expiry";
 public const string CACHE_CAPACITY_VAR = "token.cache.capacity";
+
+public const string DOCKER_REGISTRY_SERVICE_NAME = "Docker%20registry";
```

Comment:
Can we URL encode this on demand ? It's not nice to keep this constant value URL encoded.